Explorer++/ShellBrowser/SortManager.cpp

```diff
@@ -65,11 +65,11 @@ int CALLBACK CShellBrowser::Sort(int InternalIndex1,int InternalIndex2) const
 	
 	/* Folders will always be sorted separately from files,
 	except in the recycle bin. */

```

Comment:
Update this comment so that it mentions folders will only be sorted separately if the appropriate option isn't set.